lua/nvim-lsp-installer/servers/jdtls/init.lua

```diff
@@ -10,6 +10,7 @@ return function(name, root_dir)
         local executable = vim.env.JAVA_HOME and path.concat { vim.env.JAVA_HOME, "bin", "java" } or "java"
         local jar = vim.fn.expand(path.concat { root_dir, "plugins", "org.eclipse.equinox.launcher_*.jar" })
         local lombok = vim.fn.expand(path.concat { root_dir, "lombok.jar" })
+        local workspace_dir = vim.fn.fnamemodify(vim.fn.getcwd(), ":p:h:t")
```

Comment:
Wouldn't the following do it?

```suggestion
        local workspace_dir = vim.fn.fnamemodify(vim.fn.getcwd(), ":t")
```

Reply:
Not sure, nvim-jdtls decided to do it this way and I trusted them: https://github.com/mfussenegger/nvim-jdtls